---------------------------------------
Begin Slurm Prolog: Apr-03-2024 15:14:33
Job ID:    5518653
User ID:   shardin31
Account:   gts-jw254-coda20
Job name:  Slurmshardinpowderdayrun
Partition: cpu-small
QOS:       inferno
---------------------------------------

CommandNotFoundError: Your shell has not been properly configured to use 'conda deactivate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.



Lmod is automatically replacing "mvapich2/2.3.6-ouywal" with "openmpi/4.1.4".

yt : [INFO     ] 2024-04-03 15:15:50,762 Global parallel computation enabled: 5 / 48
yt : [INFO     ] 2024-04-03 15:15:50,764 Global parallel computation enabled: 24 / 48
yt : [INFO     ] 2024-04-03 15:15:50,762 Global parallel computation enabled: 7 / 48
yt : [INFO     ] 2024-04-03 15:15:50,762 Global parallel computation enabled: 12 / 48
yt : [INFO     ] 2024-04-03 15:15:50,762 Global parallel computation enabled: 16 / 48
yt : [INFO     ] 2024-04-03 15:15:50,762 Global parallel computation enabled: 18 / 48
yt : [INFO     ] 2024-04-03 15:15:50,762 Global parallel computation enabled: 22 / 48
yt : [INFO     ] 2024-04-03 15:15:50,762 Global parallel computation enabled: 10 / 48
yt : [INFO     ] 2024-04-03 15:15:50,763 Global parallel computation enabled: 1 / 48
yt : [INFO     ] 2024-04-03 15:15:50,763 Global parallel computation enabled: 4 / 48
yt : [INFO     ] 2024-04-03 15:15:50,763 Global parallel computation enabled: 17 / 48
yt : [INFO     ] 2024-04-03 15:15:50,764 Global parallel computation enabled: 19 / 48
yt : [INFO     ] 2024-04-03 15:15:50,765 Global parallel computation enabled: 2 / 48
yt : [INFO     ] 2024-04-03 15:15:50,765 Global parallel computation enabled: 11 / 48
yt : [INFO     ] 2024-04-03 15:15:50,765 Global parallel computation enabled: 9 / 48
yt : [INFO     ] 2024-04-03 15:15:50,765 Global parallel computation enabled: 3 / 48
yt : [INFO     ] 2024-04-03 15:15:50,765 Global parallel computation enabled: 21 / 48
yt : [INFO     ] 2024-04-03 15:15:50,766 Global parallel computation enabled: 14 / 48
yt : [INFO     ] 2024-04-03 15:15:50,767 Global parallel computation enabled: 20 / 48
yt : [INFO     ] 2024-04-03 15:15:50,767 Global parallel computation enabled: 15 / 48
yt : [INFO     ] 2024-04-03 15:15:50,776 Global parallel computation enabled: 13 / 48
yt : [INFO     ] 2024-04-03 15:15:50,777 Global parallel computation enabled: 8 / 48
yt : [INFO     ] 2024-04-03 15:15:50,778 Global parallel computation enabled: 6 / 48
yt : [INFO     ] 2024-04-03 15:15:50,778 Global parallel computation enabled: 0 / 48
yt : [INFO     ] 2024-04-03 15:15:50,778 Global parallel computation enabled: 23 / 48
yt : [INFO     ] 2024-04-03 15:15:50,764 Global parallel computation enabled: 25 / 48
yt : [INFO     ] 2024-04-03 15:15:50,765 Global parallel computation enabled: 41 / 48
yt : [INFO     ] 2024-04-03 15:15:50,765 Global parallel computation enabled: 45 / 48
yt : [INFO     ] 2024-04-03 15:15:50,766 Global parallel computation enabled: 32 / 48
yt : [INFO     ] 2024-04-03 15:15:50,767 Global parallel computation enabled: 42 / 48
yt : [INFO     ] 2024-04-03 15:15:50,768 Global parallel computation enabled: 37 / 48
yt : [INFO     ] 2024-04-03 15:15:50,768 Global parallel computation enabled: 29 / 48
yt : [INFO     ] 2024-04-03 15:15:50,769 Global parallel computation enabled: 35 / 48
yt : [INFO     ] 2024-04-03 15:15:50,769 Global parallel computation enabled: 38 / 48
yt : [INFO     ] 2024-04-03 15:15:50,769 Global parallel computation enabled: 36 / 48
yt : [INFO     ] 2024-04-03 15:15:50,770 Global parallel computation enabled: 40 / 48
yt : [INFO     ] 2024-04-03 15:15:50,770 Global parallel computation enabled: 39 / 48
yt : [INFO     ] 2024-04-03 15:15:50,770 Global parallel computation enabled: 27 / 48
yt : [INFO     ] 2024-04-03 15:15:50,770 Global parallel computation enabled: 28 / 48
yt : [INFO     ] 2024-04-03 15:15:50,772 Global parallel computation enabled: 30 / 48
yt : [INFO     ] 2024-04-03 15:15:50,772 Global parallel computation enabled: 46 / 48
yt : [INFO     ] 2024-04-03 15:15:50,772 Global parallel computation enabled: 31 / 48
yt : [INFO     ] 2024-04-03 15:15:50,772 Global parallel computation enabled: 44 / 48
yt : [INFO     ] 2024-04-03 15:15:50,773 Global parallel computation enabled: 43 / 48
yt : [INFO     ] 2024-04-03 15:15:50,781 Global parallel computation enabled: 33 / 48
yt : [INFO     ] 2024-04-03 15:15:50,782 Global parallel computation enabled: 26 / 48
yt : [INFO     ] 2024-04-03 15:15:50,782 Global parallel computation enabled: 47 / 48
yt : [INFO     ] 2024-04-03 15:15:50,783 Global parallel computation enabled: 34 / 48
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
Additional features and improved performance (usually) by saving this arbor with "save_arbor" and reloading:
	>>> a = ytree.load("/storage/home/hcoda1/0/jw254/data/SG64-2020/rockstar_halos-jhw/trees/tree_0_0_0.dat")
	>>> fn = a.save_arbor()
	>>> a = ytree.load(fn)
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P029 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P040 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P036 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P043 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P034 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P046 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P037 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P025 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P042 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P044 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P045 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P027 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
P029 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 29.
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P031 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P035 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P024 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P026 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P030 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P032 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
P036 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 36.
P040 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 40.
P043 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 43.
P034 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 34.
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P041 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
P046 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 46.
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P028 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P033 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
P037 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 37.
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P038 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P047 yt : [ERROR    ] 2024-04-03 15:15:50,906 NameError: name 'outputs' is not defined
P025 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 25.
P042 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 42.
P044 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 44.
P045 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 45.
P027 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 27.
P031 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 31.
P035 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 35.
P030 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 30.
P024 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 24.
P032 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 32.
P026 yt : [ERROR    ] 2024-04-03 15:15:50,906 Error occurred on rank 26.
P041 yt : [ERROR    ] 2024-04-03 15:15:50,907 Error occurred on rank 41.
P033 yt : [ERROR    ] 2024-04-03 15:15:50,907 Error occurred on rank 33.
P047 yt : [ERROR    ] 2024-04-03 15:15:50,907 Error occurred on rank 47.
P028 yt : [ERROR    ] 2024-04-03 15:15:50,907 Error occurred on rank 28.
P038 yt : [ERROR    ] 2024-04-03 15:15:50,907 Error occurred on rank 38.
  File "stellar_mass.py", line 92, in <module>
    ap.add_recipe(stellar_mass_recipe)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_pipeline.py", line 140, in add_recipe
    recipe(self)
  File "/storage/home/hcoda1/7/shardin31/.conda/envs/pd_environment_3/lib/python3.8/site-packages/ytree/analysis/analysis_operators.py", line 30, in __call__
    return self.function(target, *self.args, **self.kwargs)
  File "stellar_mass.py", line 52, in stellar_mass_recipe
    pipeline.add_operation(get_yt_dataset, outputs)
P039 yt : [ERROR    ] 2024-04-03 15:15:50,907 NameError: name 'outputs' is not defined
P039 yt : [ERROR    ] 2024-04-03 15:15:50,907 Error occurred on rank 39.
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 29 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 37 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 42 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 46 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 24 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 26 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 32 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 25 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 47 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 28 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 33 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 38 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 39 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 36 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 40 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 27 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 31 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 35 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 43 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 45 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 34 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 44 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 30 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 41 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
slurmstepd: error: *** STEP 5518653.0 ON atl1-1-02-020-33-2 CANCELLED AT 2024-04-03T15:15:50 ***
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
srun: error: atl1-1-02-020-34-2: tasks 24-47: Killed
srun: error: atl1-1-02-020-33-2: tasks 0-23: Killed
---------------------------------------
Begin Slurm Epilog: Apr-03-2024 15:15:52
Job ID:        5518653
Array Job ID:  _4294967294
User ID:       shardin31
Account:       gts-jw254-coda20
Job name:      Slurmshardinpowderdayrun
Resources:     cpu=48,mem=100G,node=2
Rsrc Used:     cput=01:04:00,vmem=5920K,walltime=00:01:20,mem=5912K,energy_used=0
Partition:     cpu-small
QOS:           inferno
Nodes:         atl1-1-02-020-33-2,atl1-1-02-020-34-2
---------------------------------------
